pxr/usd/sdf/testenv/testSdfPathExpressionArray.py

```diff
@@ -0,0 +1,32 @@
+#!/pxrpythonsubst
+#
+# Copyright 2024 Pixar
```

Comment:
2024 -> 2025

Reply:
To be fair, it was 2024 when I submitted this PR :)